docs/TESTING.md

```diff
@@ -0,0 +1,150 @@
+---
+For release: v1.0
+---
+
+# Test Plan
+
+## Introduction
+
+The idea of this The Test Plan is to communicate the test approach to team members. 
+
+As the team works with the product, they will define the needs for the vNext.
+
+### Team communication
+
+Perform communication through Slack and Github issues.
+
+### Objectives
+
+The 1.0 phase will focus on manual testing for now, and then we'll migrate to automation in the future. Preferably in the next major test plan update, the next app release will follow.
+
+App milestones for 1.0:  
+[https://github.com/WorldHealthOrganization/app/milestone/4](https://github.com/WorldHealthOrganization/app/milestone/4)
+
+### Application architecture
+
+Application is build using Flutter framework. We need to have this in mind and choose appropriate tools and approach.
+
+We're supporting Android and iOS mobile operating systems.
+
+* IOS from 8 to 13
+* Android from 4.1\(API 16\) to 10.0 \(API 29\);
+
+## Plan
+
+### Manual tests
+
+#### Scope
+
+In manual testing, we should focus on all user interface elements, hand washing animation in "Protect Yourself".
+
+#### Examples of configurations to focus on;
+
+* Android GO.
+* Older devices such as Galaxy S3 and iPhone 4s.
+* Smaller screen devices - iPhone 5.
+* Big screens - big tablets such as SM-T670 18.4".
+* Big screen with smaller resolution - iPhone XR.
+* Xiaomi devices.
+
+### Automated checks
+
+We plan to use Firebase Robo test checks for now, and add UI integration tests in the second phase.
+
+We can also use cloud device labs as support in performing various test types.
+
+### Compatibility with Device Hardware
+
+#### Different screen sizes and resolutions;
+
+* The app scales all user interface elements according to current screen density and size.
+* User interface elements do not overlap.
+* Usability or touch issues do not occur. 
+* There is no problematic shrinkage of images because of high DPI/PPI.
+
+#### App permissions;
+
+* The app can work with reduced permissions. It asks the user to grant these permissions and does not fail in an unexplained manner.
+* Permissions are only requested for the resources which are relevant to the app’s functionality.
+* The app functionality responds correctly if permission is withdrawn or rejected.
+
+### Compatibility with Device Software
+
+#### Sending, receiving and opening app notifications;
+
+* The correct handling of notifications received when the app is in the foreground or background, especially under low battery conditions.
+
+#### User Preferences
+
+* Users can amend typical preference options such as sound, brightness, network, power save mode, date and time, time zone, languages, access type and notifications.
+* The apps adhere to the set preferences by behaving accordingly.
+
+#### Interoperability
+
+* Various system overlays made by manufacturers
+  * Samsung TouchWiz;
```

Comment:
How was this list chosen?  Is it the most prevalent?

Reply:
To the best of your knowledge, those are the most popular Android overlays. The list is not, of course, fixed, it would be almost impossible to list all of them.